node/core/approval-voting/src/criteria.rs

```diff
@@ -466,12 +602,12 @@ pub(crate) enum InvalidAssignmentReason {
 /// This function does not check whether the core is actually a valid assignment or not. That should be done
 /// outside the scope of this function.
 pub(crate) fn check_assignment_cert(
-	claimed_core_index: CoreIndex,
+	claimed_core_indices: Vec<CoreIndex>,
```

Comment:
How can claimed_core_indices be an input to this function?  Aka how can whatever code invokes this function know what core it's looking for?